### PR TITLE
Add descriptions to administrative monitors

### PR DESCRIPTION
**Pull request by daniel-beck**
https://github.com/jenkinsci/jenkins/pull/5923#pullrequestreview-810988422 reminded me about this feature which I added in #2546 (which has a "before" screenshot) but never made use of.

I still think it's useful to have a short explanation what each admin monitor does in case the display name is not useful enough, so this adds descriptions to all admin monitors in core. Alternatively, we could look into making these `(?)` help instead, which would support a more extensive description. (But this could probably be done later and not even require a rename of these files, being compatible with any external-to-core admin monitors that currently have a description.)

In terms of phrasing, I tried to not use the term "administrative monitor" based on an old conversation I had with Tyler or Jesse (unsure) who said the term is confusing to admins, so a lot of "this" or just no subject at all in the descriptions.

<details>
<summary>Screenshot</summary>

> Screenshot 2021-11-20 at 15 03 02

</details>

One downside is that this drowns out the "Security" labels a bit. Thoughts?

### Proposed changelog entries

* Add descriptions of built-in administrative alerts to the global configuration alert selection page.

### Proposed upgrade guidelines

N/A

### Submitter checklist

- [ ] (If applicable) Jira issue is well described
- [ ] Changelog entries and upgrade guidelines are appropriate for the audience affected by the change (users or developer, depending on the change). [Examples](https://github.com/jenkins-infra/jenkins.io/blob/master/content/_data/changelogs/weekly.yml)
  * Fill-in the `Proposed changelog entries` section only if there are breaking changes or other changes which may require extra steps from users during the upgrade
- [ ] Appropriate autotests or explanation to why this change has no tests
- [ ] For dependency updates: links to external changelogs and, if possible, full diffs

### Desired reviewers

@mention

### Maintainer checklist

Before the changes are marked as `ready-for-merge`: 

- [ ] There are at least 2 approvals for the pull request and no outstanding requests for change
- [ ] Conversations in the pull request are over OR it is explicit that a reviewer does not block the change
- [ ] Changelog entries in the PR title and/or `Proposed changelog entries` are correct
- [ ] Proper changelog labels are set so that the changelog can be generated automatically
- [ ] If the change needs additional upgrade steps from users, `upgrade-guide-needed` label is set and there is a `Proposed upgrade guidelines` section in the PR title. ([example](https://github.com/jenkinsci/jenkins/pull/4387))
- [ ] If it would make sense to backport the change to LTS, a Jira issue must exist, be a _Bug_ or _Improvement_, and be labeled as `lts-candidate` to be considered (see [query](https://issues.jenkins-ci.org/issues/?filter=12146)).

### CODE REVIEW

File: core/src/main/resources/jenkins/security/csrf/CSRFAdministrativeMonitor/description.properties
Comment (Wadeck) [inline core/src/main/resources/jenkins/security/csrf/CSRFAdministrativeMonitor/description.properties:1]: Is it working without the use of `j:out`? (not manally tested)

**Review by Wadeck — APPROVED** [review]
Nice idea 👍

**Review by basil — APPROVED** [review]
This PR is now ready for merge. We will merge it after approximately 24 hours if there is no negative feedback. Please see the [merge process documentation](https://github.com/jenkinsci/jenkins/blob/master/docs/MAINTAINERS.adoc#merge-process) for more information about the merge process. Thanks!

File: core/src/main/resources/jenkins/security/csrf/CSRFAdministrativeMonitor/description.properties
Comment (daniel-beck) [reply core/src/main/resources/jenkins/security/csrf/CSRFAdministrativeMonitor/description.properties:1]: See screenshot.

At least in Jelly views, parameters to localized expressions are escaped, but content is not.

**Pull request merged**
